mongodb_cmdline_tool/setupenv.py

```diff
@@ -293,7 +293,7 @@ def macos_extra(c):
         # Ignore errors since ninja may already exist.
         c.run('git clone https://github.com/RedBeard0531/mongo_module_ninja ninja', warn=True)
     with c.cd(str(kHome / 'mongo')):
-        ninja_cmd = 'python buildscripts/scons.py CC=clang CXX=clang++ '
+        ninja_cmd = 'python2 buildscripts/scons.py CC=clang CXX=clang++ '
```

Comment:
I believe this will still fail because scons needs to install its dependencies in `etc/pip/dev-requirements.txt`. It's fine to keep your change for now since it's still an improvement.

Reply:
It's possible I've done something weird in my config, but I'm pretty sure this all works now.